leapp/cli/upgrade/__init__.py

```diff
@@ -16,6 +16,7 @@
 from leapp.utils.clicmd import command, command_opt
 from leapp.utils.output import report_errors, report_info, beautify_actor_exception
 from leapp.utils.report import fetch_upgrade_report_raw
+import leapp.reporting
```

Comment:
you will probably have to put here a pylint disable due to unused import

Reply:
ok no :D

Reply:
:D Does the pyling check of unused imports work? :D If yes, it has good heaurestic for `__init__.py` files.